installer/scripts/tomlparser.rb

```diff
@@ -1,6 +1,13 @@
 #!/usr/local/bin/ruby
 
-require_relative "tomlrb"
+#this should be require relative in Linux and require in windows, since it is a gem install on windows
+@os_type = ENV["OS_TYPE"]
+if !@os_type.nil? && !@os_type.empty? && @os_type.strip.casecmp("windows") == 0
+  require "tomlrb"
```

Comment:
pin to a specific version

Reply:
This is done in Docker-Provider : basimage/Dockerfile